deployment/bin/backup-rds

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# backup-data-volume /dev/xvdr
+
+RDS_ID=$(wget -q -O - http://instance-data/latest/user-data | grep 'RDS_ID=')
```

Comment:
So. it turns out that `instance-data` exists as a DNS entry in the amazon internal DNS zone `ec2.internal`:

```
ubuntu@ip-172-31-27-138:~$ dig instance-data.ec2.internal

; <<>> DiG 9.9.5-3ubuntu0.2-Ubuntu <<>> instance-data.ec2.internal
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 13536
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;instance-data.ec2.internal.	IN	A

;; ANSWER SECTION:
instance-data.ec2.internal. 20	IN	A	169.254.169.254

;; Query time: 1 msec
;; SERVER: 172.31.0.2#53(172.31.0.2)
;; WHEN: Fri Oct 07 12:05:25 UTC 2016
;; MSG SIZE  rcvd: 71
```

That's why the `instance-data` hostname works.

So yes it works, but I am surprised because I did not know that (I cut and pasted the `wget`).